playground/backend/internal/fs_tool/fs.go

```diff
@@ -29,15 +29,15 @@ const (
 )
 
 // Folder contains names of folders with executable and compiled files.
-// For each SDK these values should be set depending on folders that need for the SDK (/src and /bin for java SDK).
+// For each SDK these values should be set depending on folders that need for the SDK.
```

Comment:
can we keep the /src and /bin for java comment? This helps readers get an idea of what the `Folder` struct holds - otherwise it's hard to tell.

Reply:
will be returned!